hack/build/etcd-backup-operator/build

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+
+function go_build {
+	echo "building "${1}"..."
+	if [ ! -z ${GOINSTALL+x} ] && [ "${GOINSTALL}" = "y" ]
+	then
+  		GOBIN=${bin_dir} GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go install -installsuffix cgo  -ldflags "$go_ldflags" ./cmd/${1}/
+		mv ${bin_dir}/${1} ${bin_dir}/etcd-backup-${1}
+	else
+  		GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o ${bin_dir}/etcd-${1} -installsuffix cgo  -ldflags "$go_ldflags" ./cmd/${1}/
+	fi
+}
+
+if ! which go > /dev/null; then
+	echo "golang needs to be installed"
+	exit 1
+fi
+
+GIT_SHA=`git rev-parse --short HEAD || echo "GitNotFound"`
+
+gitHash="github.com/coreos/etcd-operator/version/etcd-backup-operator.GitSHA=${GIT_SHA}"
+
+go_ldflags="-X ${gitHash}"
+
+bin_dir="$(pwd)/_backup_output/bin"
```

Comment:
Don't use `_backup_output`. Just use `_output`.

Reply:
We should follow current etcd operator conventions as much as possible instead of creating new conventions

Reply:
no, because when i build the backup-operator image, i don't want to copy operator binary if there is any.

Reply:
> i don't want to copy operator binary if there is any.

Don't mix your personal problem and for your personal convenience into ant open source, collaborative projects.
There are rules and conventions to follow for contributions.

Reply:
if I build operator first, then i decided to build backup operator. if using `_output` for both build, then building docker image for ` backup operator` will contain operator binary right?

Reply:
Which Dockerfile?

Reply:
https://github.com/coreos/etcd-operator/pull/1458/files#diff-5a044be9f0e2ae8abe9b614f59232eb0

Reply:
there are two docker files, one for `build/etcd-backup-operator/Dockerfile` and another one for `build/operator/Dockerfile`

Reply:
Just change the PR to include this specific binary instead.